apps/web/pages/team/[slug].tsx

```diff
@@ -281,9 +273,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     isTeamView: true,
     isOrgView: isValidOrgDomain && context.resolvedUrl === "/",
   });
-  const ssr = await ssrInit(context, {
-    noI18nPreload: isValidOrgDomain && !team?.parent,
-  });
+  const ssr = await ssrInit(context);
```

Comment:
Removed option to not preload i18n introduced recently as 99% of the extra page size came from logo not i18n